### ISSUE TITLE
Accessibility props are not supported

### ISSUE THREAD

**Issue by janlonden** [yujong-lee/react-native-switch-toggle#53]
**Describe the bug**
The component doesn't support accessibility props such as `accessibilityLabel` and `accessibilityState`.

**Comment by yujong-lee** [comment]
Yes. Any PR related to this are welcomed. Thanks for the report.
